src/quacc/atoms/skzcam.py

```diff
@@ -6,35 +6,173 @@
 import numpy as np
 from ase.atoms import Atoms
 from ase.data import atomic_numbers
-from ase.io import write
+from ase.io import read, write
 from ase.units import Bohr
 
+try:
+    from chemsh import *
```

Comment:
Using `*` imports is not ideal programming practice. Let's replace this with the direct imports. 👍